servers/src/main/java/tachyon/master/file/meta/InodeFile.java

```diff
@@ -31,6 +33,9 @@
  * Tachyon file system's file representation in the file system master.
  */
 public final class InodeFile extends Inode {
+  /** The default permission for file is used the directory permission umasked 0111 */
+  private static final FsPermission SUMASK = new FsPermission((short) 0111);
```

Comment:
Hi @ooq , I use the constants configuration. I think it doesn't need to change it configurable. The default permission is the same as the Linux filesystem or HDFS system.